src/taipy/core/task/_task_manager.py

```diff
@@ -140,6 +140,12 @@ def _get_children_entity_ids(cls, task: Task):
                 entity_ids.job_ids.add(job.id)
         return entity_ids
 
+    @classmethod
+    def _is_submittable(cls, task: Union[Task, TaskId]) -> bool:
+        if isinstance(task, str):
+            task = cls._get(task)
+        return True if isinstance(task, Task) else False
```

Comment:
```suggestion
        return isinstance(task, Task)
```
Same comment as above